lib/caseStatus.ts

```diff
@@ -7,9 +7,11 @@ const headers = { 'x-api-key': AWS_KEY };
 export const getCaseStatusByPersonId = async (
   personId: number
 ): Promise<PersonCaseStatus> => {
-  const { data, error }: { data: PersonCaseStatus; error: Error } =
-    await axios.get(`${ENDPOINT_API}/residents/${personId}/casestatuses`, {
+  const { data }: { data: PersonCaseStatus; error: Error } = await axios.get(
```

Comment:
Should instead of type `Error` should it be `AxiosError`?

Reply:
Also do we even need Error?

could we just write 

`const { data } = await axios.get<PersonCaseStatus>(
    `${ENDPOINT_API}/residents/${personId}/casestatuses`,
    {
      headers,
    }
  );`

Reply:
Not really needed, will remove it!